laravel-app/public/assets/js/likeUnlike.js

```diff
@@ -50,19 +50,20 @@ function handleLikeUnlike(postId, action, element) {
 
     
     setTimeout(function () {
```

Comment:
what is the setTimeout function here?

Reply:
I used setTimeout function in like/unlike button sir to avoid button spams and delay the submission of the button, during which the button is unclickable.

Reply:
hmmmm, is this working?? can't you use debouncing instead.

Reply:
oh, will research about debouncing sir

Reply:
done sir, used throttle for like and follow buttons to avoid spam. Thank you